### PR TITLE
[move-prover] Added the "tee" command to see the full log

### PR DESCRIPTION
**Pull request by junkil-park**
- Added the "tee" command to see the full log of the Prover's inconsistency test

## Motivation

To see the full log of the Prover's daily test

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

yes

### CODE REVIEW

**Review by rexhoffman — APPROVED** [review]

**Comment by junkil-park** [comment]
Ready to review again. "set -o pipefail" is needed for `tee` to relay the error code. That command only works within `bash`.

File: .github/workflows/daily.yml [hunk 1]
```diff
@@ -199,10 +199,13 @@ jobs:
...
       - name: Run the prover tests with the inconsistency check and other nightly checks
+        shell: bash
         run: |
           cd /opt/git/diem/
-          MVP_TEST_INCONSISTENCY=1 cargo test -p move-prover --release >> $MESSAGE_PAYLOAD_FILE
-          MVP_TEST_FEATURE=cvc4 cargo test -p move-prover --release >> $MESSAGE_PAYLOAD_FILE
+          set -o pipefail
+          MVP_TEST_INCONSISTENCY=1 cargo test -p move-prover --release 2>&1 | tee -a $MESSAGE_PAYLOAD_FILE
+          MVP_TEST_FEATURE=cvc4 cargo test -p move-prover --release 2>&1 | tee -a $MESSAGE_PAYLOAD_FILE
+
```
Comment (sausagee) [inline .github/workflows/daily.yml:208]: nit - extra blank line

**Review by sausagee — COMMENTED** [review]
Looks good other than the nit.  Will stamp after fixing.

File: .github/workflows/daily.yml (hunk 1 above)
Comment (junkil-park) [reply .github/workflows/daily.yml]: Done.

**Review by sausagee — DISMISSED** [review]

**Comment by junkil-park** [comment]
ci-test keeps on failing on "docker-compose test" and "helm test". Is there anything I should fix? Or, should I keep trying to land?

**Pull request closed without merging**